src/librustc_back/target/windows_base.rs

```diff
@@ -64,8 +64,7 @@ pub fn opts() -> TargetOptions {
     ]);
 
     TargetOptions {
-        // FIXME(#13846) this should be enabled for windows
-        function_sections: false,
+        function_sections: true,
```

Comment:
Is this supposed to be changed to `true`?

Reply:
Yes, I tested this locally and it links successfully with `x86_64-pc-windows-gnu`. The FIXME was about turning on the flag once it is no longer an issue.